.github/workflows/ci.yml

```diff
@@ -8,6 +8,8 @@ on:
     tags:
       - "v[0-9]+.[0-9]+.[0-9]+*"
   pull_request:
+    branches:
+      - "*"
```

Comment:
Does it complain if we don't specify this?

Reply:
yeah it complains about "empty value in block mapping"